### PR TITLE
Wait for 2FA to complete

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
## Description

**Fixes** #1 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by rajkumaar23** [comment]
@Nancy-Chauhan I guess this fixes #19 as well ?

**Comment by Nancy-Chauhan** [comment]
Yes @rajkumaar23 . This will fix the issue

File: scrape.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ const linkedinLogin = async (username, password, page) => {
 
   // Wait for page load
   await page.waitForNavigation();
```
Comment (ayushjainrksh) [inline scrape.js:18]: This line isn't needed anymore and would be a problem for users without 2FA as the page doesn't navigate.

**Review by ayushjainrksh — CHANGES_REQUESTED** [review]

**Comment by ayushjainrksh** [comment]
Thanks for the PR @Nancy-Chauhan. LGTM except for a small change.

**Pull request merged**